docs/guide/docs/releases/v12_10_0.md

```diff
@@ -0,0 +1,44 @@
+---
+id: v12_10_0
+title: Release Notes - v12.10.0
+---
+
+## Highlights
+
+- [Module Upload](#module-upload) - Upload custom modules via the Administrator interface
+- [New QnA Interface](#new-qna-interface) - Brand new interface for the QnA module
+- [Enterprise Branding](#enterprise-branding) - Easier configuration to white-label Botpress
+- [Faster NLU Training](#faster-nlu-training) - New way to treat out of scope data
+- [Broadcast Module](#broadcast-module) - \*\* Experimental
+
+### Module Upload
+
+Modules gives a lot of freedom to customize Botpress and add features, however it's a bit complicated to manage and update them. This release adds a new button on the Modules management page (only accessible to Super Admins), to upload module archive on the server.
+
+Upon server restart, custom modules are loaded in priority (before built-in modules) so you can easily replace any module.
+
+This also work if you are running Botpress on a cluster.
+
+### New QnA Interface
+
+We are working on a brand new interface for the Studio and we decided to rewrite the QnA module interface from the ground up. We are removing most modals to make working with Botpress more efficient.
+
+![Interface](assets/releases/v12_10_0/qnaInterface.png)
+
+### Enterprise Branding
```

Comment:
In terms of backwards compatibility, is there any work to be done by already white-labeled accounts?